copper-server-impl/src/main/java/com/pogeyan/cmis/impl/services/CmisTypeServices.java

```diff
@@ -1973,13 +1986,17 @@ public static DocumentTypeDefinition getDocumentDefinitionWithTypePermission(
 
 			return typeDef;
 		}
+	}
+
+	public static Boolean checkCrudPermission(ITypePermissionService typePermissionFlow, String repositoryId,
+			IUserObject role, String typeId, TypePermissionType permission) {
```

Comment:
Can we have an array of permissions passed? [TypePermission.Create, TypePermission.Update]